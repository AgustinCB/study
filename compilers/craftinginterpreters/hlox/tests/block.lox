print "BEFORE";
{
  print "ON THE BLOCK";
  var a;
  print a;
  print a=3;
  print a+2;
}
print "AFTER";