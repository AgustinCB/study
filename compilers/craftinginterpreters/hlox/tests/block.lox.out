BEFORE
ON THE BLOCK
ASDASD
AFTER